agent/__init__.py

```diff
@@ -642,6 +648,29 @@ def write_metadata(data, rewrite_file):
     metadata_path.chmod(0o644)
 
 
+def enroll_device(enroll_token, claim_token, device_id):
+
+    payload = {
+        'key': enroll_token,
+        'claim_token': claim_token,
+        'device_id': device_id
+    }
+
+    try:
+        enroll_req = requests.post(
+            '{}/v0.2/enroll-device'.format(WOTT_ENDPOINT),
+            json=payload
+        )
+    except requests.exceptions.RequestException:
```

Comment:
why don't you write the error details to the log?

Reply:
> exception(msg, *args, **kwargs)¶
Logs a message with level ERROR on this logger. The arguments are interpreted as for debug(). Exception info is added to the logging message. This method should only be called from an exception handler.

using the `logger.exception()` function in this sense is the same as including the `exc_info=True` parameter in the `logger.error()`. 

P.S. I added more suitable error handling function on `not  request.ok`. But here, what else over the exception info, you want to see in log at this place?

Reply:
> using the `logger.exception()` function in this sense is the same as including the `exc_info=True` parameter in the `logger.error()`.

ok, thanks for clarification